sql/core/src/main/scala/org/apache/spark/sql/DataFrame.scala

```diff
@@ -730,6 +731,23 @@ class DataFrame private[sql](
     Generate(generator, join = true, outer = false, None, logicalPlan)
   }
 
+  /**
+   * Returns a new [[DataFrame]] that omits rows with `NULL` values in the specified columns.
+   * If no columns are specified, a row is omitted if any column contains `NULL`.
+   *
+   * @group dfops
+   */
+  @scala.annotation.varargs
+  def dropna(cols: String*): DataFrame = {
```

Comment:
I'd vote for `dropna` simply to be consistent with the Pandas API which we have been using as a guide so far. I don't mind having both `dropNull` and `dropna`